src/controllers/__tests__/events.test.js

```diff
@@ -150,4 +150,26 @@ describe("Testing events for routes doesn't require auth controls", () => {
         return events;
       });
   });
+
+  it('POST /events/:id/pending-applicants/:userId/decline should decline an applicant', (done) => {
+    request(app)
+      .post(`/api/events/1/pending-applicants/1/decline`)
+      .send(events[0])
+      .expect('Content-Type', /json/)
+      .expect(200, (err, res) => {
+        if (err) {
+          done();
+          return err;
+        }
+        if (!res.body) {
+          expect(res.statusCode).toBe(404);
+          done();
+          return res.json('No event found');
+        }
+        expect(res.body.data).toEqual(events[0]);
+        expect(typeof res.body).toEqual('object');
+        done();
+        return events;
+      });
+  });
```

Comment:
Hafız detected that this test always passes so we need to change it a bit.

Reply:
Thank you @dilarafirtina for all your comments, and this one especially cuz it is crucial.